relay-server/src/services/spooler/spool_utils.rs

```diff
@@ -0,0 +1,34 @@
+//! Contains helper utils which help to manage the spooler and spooled data.
+
+use std::path::PathBuf;
+
+use sqlx::sqlite::{SqliteConnectOptions, SqliteJournalMode, SqlitePoolOptions};
+
+use crate::service::create_runtime;
+use crate::services::spooler::{sql, BufferError};
+
+/// Truncates the spool file deleting all the persisted on-disk data.
+pub fn truncate(path: PathBuf) -> Result<(), BufferError> {
+    let options = SqliteConnectOptions::new()
+        .filename(path)
+        .journal_mode(SqliteJournalMode::Wal)
+        .create_if_missing(true);
```

Comment:
Shouldn't we rather error if Relay can't find the spool?

Reply:
yeah, makes sense. Will do